imports/plugins/core/checkout/server/methods/workflow.js

```diff
@@ -335,6 +335,39 @@ Meteor.methods({
     check(order, Object);
     check(itemIds, Array);
```

Comment:
This doesn't seem like the right approach to this problem. Firstly, we should really try to avoid hardcoding in individual statuses like this as it's really brittle. Also, this method is already getting an array of item Ids, it seems we should be getting the correct values there and be able to update the workflow based on those values. Basically here we are just ignoring those and rewriting the entire order items. 

Basically this is a hack around the problem rather than a real solution.

Reply:
This is a problem with the existing code as well. It's not using `itemIds` at all.

Reply:
Yes, I noticed that `itemsIds` wasn't used in the method at all also, was surprised.

Reply:
So this seems like just a slightly fancier way of hard-coding in the process. This method should not need to be aware of any specific status such as `isPack` and I don't understand why we are doing that.